spec/requests/api/users_spec.rb

```diff
@@ -85,15 +88,18 @@
       context 'パラメータが適切でない場合' do
         let(:wrong_cases) do
           [
-            { name: '', email: 'test@example.com', password: 'foobar' },
-            { name: 'hogehgoe', email: '', password: 'foobar' },
-            { name: 'hogehoge', email: 'test@example.com', password: '' },
+            { name: '', email: Faker::Internet.email, password: Faker::Internet.password(min_length: 6) },
+            { name: Faker::Name.name, email: '', password: Faker::Internet.password(min_length: 6) },
+            { name: Faker::Name.name, email: Faker::Internet.email, password: '' },
```

Comment:
name, email, password って let 変数を作ったほうが良さそうですね。
個別のケースに対して、ひとつずつ Faker を呼び出していくと煩雑に感じました。

現状のコードより、こちらのコードのほうがエラーパターンが明確だと思いませんか。
```suggestion
            { name: '', email:, password: },
            { name:, email: '', password: },
            { name:, email:, password: '' },
```

※ let 変数の定義は省略して書いてます


今の書き方が悪いというわけではないのですが、情報量が多すぎて、各 case が何をしているのかが読みづらくなっているのは厳しいです。
rspec は「動く仕様書」であり、内容が読みやすいことも期待値のひとつにあります。